camel/sync/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,4 +35,9 @@
     <routeContextRef ref="drupalAuthentication"/>
   </camelContext>
 
+  <!-- configuration of activemq component -->
+  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="islandora.brokerUrl" value="${islandora.brokerUrl}"/>
```

Comment:
This should be ```<property name="brokerUrl" value="${islandora.brokerUrl}"/>```

Like this https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/blob/master/fcrepo-fixity/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L35